app/helpers/messages_helper.rb

```diff
@@ -1,2 +1,10 @@
 module MessagesHelper
+
+  def personal_address
+    unless current_user.name.blank?
+      address = current_user.name
+    else 
+      address = current_user.email
```

Comment:
What about something like `current_user.name? ? current_user.name : current_user.email` If you haven't seen this syntax before, I would research ternary operator.